src/odemis/driver/omicronxx.py

```diff
@@ -981,7 +981,6 @@ def _parseMessage(self, msg):
             self._sendAnswer("GOM", chan, "FCFB")
         elif com == "SOM":
             if len(args) == 1:
-                om = int(args[0], 16)
```

Comment:
All these lines in the omicron simulator don't do nothing. They actually check that the argument is the correct format. Arguably, you could drop the assignment, but it makes all the different cases look more similar, so I like them.